### PR TITLE
Upgrade hapi

### PR DESCRIPTION
**Pull request by paolochiodi**
builds on https://github.com/senecajs/ramanujan/pull/8

### CODE REVIEW

File: api/api-service.js [hunk 1]
```diff
@@ -9,20 +9,20 @@ var Seneca = require('seneca')
...
 })
 
 server.register({
-  register: Chairo, 
+  register: Chairo,
   options:{
     seneca: Seneca({
       tag: 'api',
       log: 'silent',
       legacy: { logging: false },
```
Comment (mcdonnelldean) [inline api/api-service.js]: don't need the legacy property in seneca 3

Comment (paolochiodi) [reply api/api-service.js]: I'll fix this in the other PR about upgrading seneca 3.
Two questions: 
1- what was it for before?
2- we may want to fix logging examples: https://github.com/senecajs/seneca/blob/master/docs/examples/custom-logger.js#L34

**Comment by mcdonnelldean** [comment]
@paolochiodi 

1: It allowed use of the new logger on master before 3 was launched. You can now just remove the line fully.

2: Noted, PR's are always appreciated though.

**Comment by paolochiodi** [comment]
Included in https://github.com/senecajs/ramanujan/pull/8

**Pull request closed without merging**